### ISSUE TITLE
Accessibility issue: Scrollable region must have keyboard access on /notebook-ci-testing/notebooks/DrizzlePac/align_sparse_fields/align_sparse_fields.html

### ISSUE THREAD

**Issue by dulude** [spacetelescope/notebook-ci-testing#288]
## What
An accessibility scan flagged 15 elements on https://spacetelescope.github.io/notebook-ci-testing/notebooks/DrizzlePac/align_sparse_fields/align_sparse_fields.html because scrollable region must have keyboard access. Learn more about why this was flagged by visiting https://dequeuniversity.com/rules/axe/4.11/scrollable-region-focusable?application=playwright.

The following elements need attention:

- `<pre id="codecell0">` (selector: `#codecell0`)
- `<pre id="codecell3">` (selector: `#codecell3`)
- `<pre id="codecell4"><span></span>Downloading URL https://mast.stsci.edu/api/v0.1/Download/file?uri=mast:HST/product/j8xi0xs0q_flc.fits to ./mastDownload/HST/j8xi0xs0q/j8xi0xs0q_flc.fits ...
</pre>` (selector: `#codecell4`)
- `<pre id="codecell6"><span></span>Downloading URL https://mast.stsci.edu/api/v0.1/Download/file?uri=mast:HST/product/j8xi0xs3q_flc.fits to ./mastDownload/HST/j8xi0xs3q/j8xi0xs3q_flc.fits ...
</pre>` (selector: `#codecell6`)
- `<pre id="codecell8"><span></span>Downloading URL https://mast.stsci.edu/api/v0.1/Download/file?uri=mast:HST/product/j8xi0xs6q_flc.fits to ./mastDownload/HST/j8xi0xs6q/j8xi0xs6q_flc.fits ...
</pre>` (selector: `#codecell8`)
- `<pre id="codecell10"><span></span>Downloading URL https://mast.stsci.edu/api/v0.1/Download/file?uri=mast:HST/product/j8xi0xsaq_flc.fits to ./mastDownload/HST/j8xi0xsaq/j8xi0xsaq_flc.fits ...
</pre>` (selector: `#codecell10`)
- `<pre id="codecell13">` (selector: `#codecell13`)
- `<table id="table140183148229328" class="table-striped table-bordered table-condensed">` (selector: `#table140183148229328`)
- `<pre id="codecell14">` (selector: `#codecell14`)
- `<table id="table140183144874896" class="table-striped table-bordered table-condensed">` (selector: `#inspect-the-alignment > .cell.container.docutils > .cell_output.container.docutils > .text_html.output > div > table`)
- `<pre id="codecell16">` (selector: `#codecell16`)
- `<pre id="codecell24">` (selector: `#codecell24`)
- `<table id="table140183144874896" class="table-striped table-bordered table-condensed">` (selector: `.cell.container.docutils:nth-child(16) > .cell_output.container.docutils > .text_html.output > div > table`)
- `<pre id="codecell38">` (selector: `#codecell38`)
- `<pre id="codecell42"><span></span>CRDS - INFO -  Symbolic context &apos;hst-latest&apos; resolves to &apos;hst_1287.pmap&apos;
CRDS - INFO -  Reorganizing 0 references from &apos;instrument&apos; to &apos;flat&apos;
CRDS - INFO -  Reorganizing from &apos;instrument&apos; to &apos;flat&apos; cache,  removing instrument directories.
</pre>` (selector: `#codecell42`)

To fix this, ensure elements that have scrollable content are accessible by keyboard in safari.

Specifically:

Fix any of the following:
-   Element should have focusable content
-   Element should be focusable

## Acceptance Criteria
- [ ] The specific violation reported in this issue is no longer reproducible.
- [ ] The fix MUST meet WCAG 2.2 guidelines OR the accessibility standards specified by the repository or organization.
- [ ] A test SHOULD be added to ensure this specific violation does not regress.
- [ ] This PR MUST NOT introduce any new accessibility issues or regressions.
